main.go

```diff
@@ -29,9 +29,12 @@ func main() {
 
 	devClient := nearapi.NewClient(externalURL)
 
+	rpcMetricCollector := collector.NewNodeRpcMetrics(client, devClient, accountID)
```

Comment:
Any reason we decide to declare/assign this variable, if we're only ever going to reference its value once? Feel like it was easier to understand before with:
```
	registry.MustRegister(
		collector.NewNodeRpcMetrics(client, devClient, accountID),
		collector.NewDevNodeRpcMetrics(devClient),
	)
```
below

Reply:
Feel free to disregard and resolve, this is a pretty subjective take

Reply:
`rpcMetricCollector` is actually used twice. Once to start the `RecordValidators` go routine. and secondly to provide the `promCollector` object